specs/waku/v2/waku-v2.md

```diff
@@ -147,7 +151,50 @@ packet from one version to the other.
 
 See [bridge spec](waku-bridge.md) for details on a bridge mode.
 
-## Changelog
+
+# Security 
+Each protocol layer of Waku v2 provides a distinct service and is associated with a separate set of security features and concerns. Therefore, the overall security of Waku v2 depends on how different layers are utilized. In this section, we overview the security properties of Waku v2 protocols against a static adversarial model which is described below. Note that a more detailed security analysis of each Waku protocol is supplied in its respective specs as well.
+
+## Adversarial Model
```

Comment:
Perhaps phrase this as "the primary adversary". This way it doesn't seem like we are completely ignoring active adversary models (spam, etc).

Reply:
Nice suggestion, done!